internal/install/recipes/bundler.go

```diff
@@ -80,9 +88,21 @@ func (b *Bundler) getBundleRecipeWithDependencies(recipe *types.OpenInstallation
 		Recipe: recipe,
 	}
 
+	// For OHIs with dual infrastructure-agent/super-agent dependencies, remove the one not needed:
+	//	if super-agent is targeted, infrastructure-agent is removed
+	//	if super-agent is not targeted, super-agent is removed
+	if recipe.IsOhi() && utils.StringInSlice(types.InfraAgentRecipeName, recipe.Dependencies) && utils.StringInSlice(types.SuperAgentRecipeName, recipe.Dependencies) && depException != "" {
```

Comment:
I don't think we need to have this IsOhi() check, this behavior can be applicable to all recipes.

Reply:
That check was removed in this [new PR](https://github.com/newrelic/newrelic-cli/pull/1536).